src/bio2bel/sources/intact.py

```diff
@@ -18,7 +18,7 @@
 import pybel.dsl
 from pybel import BELGraph
 from pybel.dsl import GeneModification, ProteinModification
-from ..utils import ensure_path
```

Comment:
stop doing that. relative imports only.

Reply:
removed it

Reply:
I know again why I did it: 
following error message occurred after I changed it again to relative import 

```python-traceback
Traceback (most recent call last):
  File "/Users/sophiakrix/git/bio2bel/src/bio2bel/sources/intact.py", line 19, in <module>
    from ..utils import ensure_path
ImportError: attempted relative import with no known parent package
```

Reply:
There's a module level docstring to remind you if you ever forget what the right way is.

Reply:
Why is  `from ..utils import ensure_path` not correct?